### PR TITLE
possible new above the fold site :)

### PR DESCRIPTION
**Pull request by bellesea**
hiiii! would appreciate general feedback + whether it's obvious that the game console render is movable AND playable (try clicking the buttons) :))

this in an implementation of the sprig poster, credits to @Krishna-bansall

### CODE REVIEW

**Comment by grymmy** [comment]
@leomcelroy I don't think this originated from an issue - please look at this and provide feedback, and remove yourself from assignees when done plz

File: src/pages/index.astro
Comment (JosiasAurel) [inline src/pages/index.astro:1]: please could you update the PR to not include formatting changes? It makes it challenging to read the diff

**Review by JosiasAurel — CHANGES_REQUESTED** [review]
besides the formatting changes, lgtm and tested 👍🏾

**Comment by leomcelroy** [comment]
I like the effort to constantly improve the site but overall I'm not a huge fan of these changes.

The main improvements I see are:

- The background makes some more sense. The original background is essentially just a dark image we liked that is pixelated.
- There is an explicit cue that the model is interactive.

The things I'm less keen on are:

- The large Sprig title is mostly obscured by the model which looks busy and haphazard.

Screenshot 2024-01-14 at 10 42 45 PM

- The explicit interactive callout blends in with the background. (refer to previous image)
- As you scroll the model obscures the text on the page.

Screenshot 2024-01-14 at 10 43 14 PM

- The new copy comes across has highly transactional, I read more emphasis on "free" rather than "craft" as it was before.
- The scroll animation reaches it's final state when the model is at the top of the page rather than the middle.

Screenshot 2024-01-14 at 10 41 56 PM

Screenshot 2024-01-14 at 10 42 26 PM

- All the green colors start to clash.

Considering all of this if we feel compelled to update this portion of the page I would maybe go ahead with finding a new background and adding some cue that you can interact with the 3D model.

**Comment by bellesea** [comment]
hey @leomcelroy thanks sm for the feedback! will try to implement! it looks like some of it is a responsive issue, which device did you check this on?

**Comment by Sheepy3** [comment]
Hi, is this PR abandoned?

**Comment by grymmy** [comment]
@bellesea do we plan on landing this?

**Comment by bellesea** [comment]
> @bellesea do we plan on landing this?

@grymmy i don't think so — wasn't able to get it reliably working on low end computers. I think we can close this for now? Thank you!

**Comment by grymmy** [comment]
@JosiasAurel I know I talked w/ you today about resurrecting this but actually @snoglobe is a good fit and will own getting this to production.

**Comment by snoglobe** [comment]
done in https://github.com/hackclub/sprig/pull/1802

**Pull request closed without merging**